scrapy_zyte_api/responses.py

```diff
@@ -188,8 +192,8 @@ def _process_response(
 
     if api_response.get("httpResponseHeaders") and api_response.get("httpResponseBody"):
         response_cls = responsetypes.from_args(
-            headers=api_response["httpResponseHeaders"],
-            url=api_response["url"],
+            headers=cast(List[Dict[str, str]], api_response["httpResponseHeaders"]),
```

Comment:
This produces an error which looks valid: `api_response["httpResponseHeaders"]` is a list of dicts with "name" and "value" keys while `ResponseTypes.from_headers` expects a `Mapping[bytes, bytes]` (not even strings).

Reply:
We’ll need to address that.